starter/js/atm.js

```diff
@@ -1,43 +1,54 @@
 //Begin with the document ready function
+$(document).ready(function(){
 
+}
     //Checking account deposit function
-
       //On click of the depositChecking button
-
+$("#depositChecking").click(function(event){
         //Get value from the amountChecking input field
-
+   var amountBalance = $("#amountChecking").val();
         //Take that value and add it to the existing value in the checkingBalance div
-
+   var checkingBalance = amountBalance.val();
     //Checking account withdrawl funtion
-
+});
+event();
       //On click of the withdrawChecking button
-
+$("#withdrawChecking").click(function(event){
         //Get value from the amountChecking input field
-
+var amountBalance = $("#amountChecking").val();
         // If that value is greater than the value in the account ignore that action
         // In other words if this would put the account into a negative balance do not allow it
-
+if (amountBalance < 0) {
+  return prompt("Not allowed!");
+} else {
+  checkingBalance - amountBalance;
+};
         //Else subtract that value from the current amount in the checking account
 
     //Savings account deposit function
 
       //On click of the depositSavings button
-
+$("#depositSavings").click(function(event){
         //Get value from the amountSavings input field
-
+var amountBalance = $("#amountSavings").val();
         //Take that value and add it to the existing value in the savingsBalance div
-
+var savingsBalance = amountBalance.val();
     //Savings account withdraw funtion
-
+});
+event();
       //On click of the withdrawl button
-
+$("#withdrawSavings").click(function(event){
         //Get value from the amountSavings input field
-
+var amountBalance = $("#amountSavings").val();
          //If that value is greater than the value in the account ignore that action
          //In other words if this would put the account into a negative balance do not allow it
 
          //Else subtract that value from the current amount in the savings account
-
+if (amountBalance < 0) {
+  return prompt("Not allowed!");
+} else {
+  savingsBalance - amountBalance;
```

Comment:
Same as above comment about need to put the result of this subtraction somewhere so you can do something with it (like print it to the screen).